src/test/java/bio/terra/cloudres/common/OperationAnnotatorTest.java

```
@@ -33,7 +33,11 @@ public void testExecuteGoogleCloudCall_success() throws Exception {
 
     operationAnnotator.executeGoogleCall(
         () -> {
-          Thread.sleep(4100);
+          try {
+            Thread.sleep(4100);
+          } catch (InterruptedException e) {
+            e.printStackTrace();
```

this should probably throw a runtime exception so that the test fails if this happens. Not very likely, but still good practice.

---

done